packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -216,8 +261,21 @@ const renderView = (displays, layout, emphasis, overrideDefaultStyling) => {
   );
 };
 
+const isDisplaysTruncated = (displays) => {
+  const displaysSlice = displays.slice(0, 8);
+
+  for (let i = 0; i < displaysSlice.length; i += 1) {
+    if (displaysSlice[i].props.isTruncated === true) {
+      return true;
+    }
+  }
+
+  return false;
+};
```

Comment:
Looks like this function is specifically used for the truncated fix. Do we want to add some basic JS unit tests on this function?

Reply:
Ok I went ahead and added a test that should cover this in it: [6fa2823](https://github.com/cerner/terra-clinical/pull/912/commits/6fa2823ad1ab1ee7520979b0f21fd904e32d92b9)

Other related test: [b97a66a](https://github.com/cerner/terra-clinical/pull/912/commits/b97a66a708c12b1e2de898b3296cc26432099523)